doc/contributing/security-release-process.md

```diff
@@ -200,6 +200,11 @@ out a better way, forward the email you receive to
   * Request publication of [H1 CVE requests][]
     * (Check that the "Version Fixed" field in the CVE is correct, and provide
       links to the release blogs in the "Public Reference" section)
+  * In case the reporter doesn't accept the disclosure follow this process:
+    * Remove the original report reference within the reference text box and
+      insert the public URL you would like to be attached to this CVE.
+    * Then uncheck the Public Disclosure on HackerOne box at the bottom of the page.
+    ![image (4)](https://github.com/RafaelGSS/node/assets/26234614/98009250-c538-4e36-895f-6d2cde4cb5c1)
```

Comment:
I don't know how GitHub assets work, but it seems odd to reference one in a non-org repository here.

Reply:
Oh, it was automatically uploaded. Let me try to adjust it.